api/swagger.yaml

```diff
@@ -8752,6 +8752,11 @@ paths:
             details.
           type: "string"
           required: true
+        - name: "platform"
+          in: "query"
+          description: "Select a platform-specific manifest to be pushed. OCI platform (JSON encoded)"
+          type: "string"
+          x-nullable: true
```

Comment:
Yeah, these are awkward to describe for query-parameters; I'll add a `docs/revisit` label to see if we can improve docs for this in a follow-up to be more explicit about what format is expected (I'll also have to look what swagger allows to define that more structured, but perhaps it will be just a description and an example).

Note to self: as part of that, describe that some fields may not be used for matching (`osfeatures`?)